servers/partner-ci/common.yaml

```diff
@@ -1,10 +1,14 @@
 - parameter:
     name: common-parameters
     parameters:
+      - string:
+          name: FUEL_RELEASE_VERSION
+          default: '9.0'
```

Comment:
Ideally, we could create
9.1 based on the fuel-qa/9.1 branch
9.x base on the fuel-qa/stable/mitaka